docs/building-modules/module-interfaces.md

```diff
@@ -174,17 +174,17 @@ Here is an example of a request handler for the `bank` module `SendReq` request
 
 The request handler can be broken down as follows:
 
-- **Parse Request:** First, it tries to parse the argument `address` into a `AccountAddress`. Then, the request handler attempts to parse the request, and then run `Sanitize` and `ValidateBasic` on the underlying `BaseReq` to check the validity of the request. Finally, it attempts to parse `BaseReq.From` to the type `AccountAddress`.
-- **Message:** Then, a [message](./messages-and-queries.md#messages) of the type `MsgSend` (defined by the module developer to trigger the state changes for this transaction) is created from the values.
-- **Generate Transaction:** Finally, the HTTP `ResponseWriter`, client `Context`, request [`BaseReq`](../interfaces/rest.md#basereq), and message is passed to `WriteGeneratedTxResponse` to further process the request.
+- **Parse Request:** First, the request handler tries to parse the argument `address` into a `AccountAddress`. The request handler then attempts to parse the request, followed by running `Sanitize` and `ValidateBasic` on the underlying `BaseReq` to check the validity of the request. Finally, the request handler attempts to parse `BaseReq.From` to the type `AccountAddress`.
+- **Message:** After parsing the request, a [message](./messages-and-queries.md#messages) of the type `MsgSend` is created from the values, which is defined by the module developer to trigger the state changes for this transaction.
+- **Generate Transaction:** Finally, the client `Context`, the HTTP `ResponseWriter`, the request's [`BaseReq`](../interfaces/rest.md#basereq), and the message are all passed to `WriteGeneratedTxResponse` to further process the request.
```

Comment:
Oh boy, this stuff is all deprecated. I bet there is a lot of stale documentation. Anyway, we can address that later

Reply:
Yea, I was not sure about a lot of this. I'd be happy to resolve in another pull request after I'm up to speed.

Reply:
Created an issue for that: https://github.com/cosmos/cosmos-sdk/issues/9283
@ryanchrypto  - feel free to assign yourself.